pkg/kubelet/network/hostport/hostport_manager_test.go

```diff
@@ -215,18 +215,18 @@ func TestGetHostportChain(t *testing.T) {
 func TestHostPortManagerRemoveLegacyRules(t *testing.T) {
```

Comment:
Remove the whole test case as we are not removing legacy rules anymore.

Reply:
Done, thanks

Reply:
Should we remove `TestHostPortSyncerRemoveLegacyRules` as well ?